apps/web-giddh/src/app/login/login.component.ts

```diff
@@ -144,9 +127,6 @@ export class LoginComponent implements OnInit, OnDestroy {
         this.isLoginWithGoogleInProcess$ = this.store.pipe(select(state => {
             return state.login.isLoginWithGoogleInProcess;
         }), takeUntil(this.destroyed$));
-        contriesWithCodes.map(c => {
```

Comment:
Please check this code should not be removed as `countryCodeList` is used to determine `selectedCountry` in method `setCountryCode`. Also `selectedCountry` is used in methods `VerifyCode` and `getOtp`